packages/client/src/__tests__/integration/happy/exhaustive-schema-mongo/__snapshots__/test.ts.snap

```diff
@@ -679,9 +679,9 @@ export class PrismaClient<
    * 
    * Read more in our [docs](https://www.prisma.io/docs/concepts/components/prisma-client/transactions).
    */
-  $transaction<P extends PrismaPromise<any>[]>(arg: [...P]): Promise<UnwrapTuple<P>>;
+  $transaction<P extends PrismaPromise<any>[]>(arg: [...P]): Promise<UnwrapTuple<P>>
```

Comment:
Note: diff int this file is mostly 3 things:

- Consistent formatting of doc comments
- Consistency regarding semicolon usage
- Consistency regarding `Foo[]` vs `Array<Foo>` type annotation.

Reply:
Wondering if we could run the generated typescript through a linter after generating... maybe not for this PR but something like this would make the style & formatting of the generated types much more consistent.

Reply:
Ideally, if we slowly migrate generator to this style in generators, our formatting would be consistent.